Libraries/dotNetRdf.Core/Query/LeviathanQueryProcessor.cs

```diff
@@ -2216,7 +2216,7 @@ private IEnumerable<SparqlQuery> GetRemoteQueries(SparqlEvaluationContext contex
         else
         {
             // Split bindings in chunks and inject them
-            foreach (ISet[] chunk in bindings.ChunkBy(100))
+            foreach (ISet[] chunk in ChunkBy(bindings, 100))
```

Comment:
That extension method had this single call site and it was not something I could replace with a single line, so I inlined it as a utility method into this file instead.